lib/layout.js

```diff
@@ -174,11 +172,21 @@ const pickZone = (opts, frameBounds, targetBounds, size) => {
     { side: `end`, standing: `below`, flow: `column`, order: 1, w: f.x2, h: (f.y2 - t.y2) },
     { side: `start`, standing: `left`, flow: `row`, order: -1, w: t.x, h: f.y2 },
   ]
-
+  
+  /* Order the zones by the amount of popup that would be cut out if that zone is used.
+     The first one in the array is the one that cuts the least amount.
+      
+     const area = size.w * size.h  // Popup area is constant and it does not change the order
+  */
+  zones.forEach((z) => {   
```

Comment:
As I wrote in the comment here: https://github.com/littlebits/react-popover/pull/60/files#diff-77ab7858fd1708d83f99faf433c11a56R213 this is actually always useful because ordering the zones by best fit. You have a few `myZones[0]` that are taking the first zone available. If we always reorder the zones by best fit, it will always pick the best fit. So for example when `opt.preferPlace="column"`, the selected zone will be the best fit between `above` and `below`. 

Considering that the `h` and `w` are already available and you have 4 zones, we are talking of a computation that is taking less than 1ms. 

If we are trying to optimize, we can actually remove the `availZones` computation, since the will be the first ones after sorting.